companies/doctolib.json

```diff
@@ -18,11 +18,11 @@
     ],
     "required-elements": [
         {
-            "desc": "email",
+            "desc": "E-Mail",
```

Comment:
We've been using `Email` without the hyphen on most other entries, not sure which one you wanted to use for consistency.

Reply:
Unfortunately, either can be correct depending on the context. :/ For German entries, we use `E-Mail` and for English entries (and ones with multiple countries), we use `Email`.

A while ago, I proposed #710 as somewhat of a solution to this annoying problem.